gcc/testsuite/jit.dg/test-tls.c

```diff
@@ -29,6 +29,8 @@ create_code (gcc_jit_context *ctxt, void *user_data)
   gcc_jit_lvalue_set_tls_model (foo, GCC_JIT_TLS_MODEL_GLOBAL_DYNAMIC);
 
   CHECK_STRING_VALUE (gcc_jit_lvalue_get_name (foo), "foo");
+  gcc_jit_lvalue_set_name (foo, "new_one");
+  CHECK_STRING_VALUE (gcc_jit_lvalue_get_name (foo), "new_one");
```

Comment:
Would it be possible to test that the asm contains the correct name?
Also, this test file is for thread-local storage, so it's probably better to move it to another file.

Reply:
Yep! Added a test for it.